config.schema.json

```diff
@@ -20,11 +23,38 @@
             "items": {
                 "$ref": "#/$defs/featured_market"
             }
+        },
+        "blacklisted_markets": {
+            "type": "array",
+            "items": {
+                "$ref": "#/$defs/blacklisted_markets"
+            }
+        },
+        "blacklisted_pools": {
+            "type": "array",
+            "items": {
+                "$ref": "#/$defs/blacklisted_pools"
+            }
+        },
+        "blacklisted_tokens": {
+            "type": "array",
+            "items": {
+                "$ref": "#/$defs/blacklisted_tokens"
+            }
         }
     },
     "$defs": {
         "featured_market": {
             "type": "string"
+        },
+        "blacklisted_markets": {
+            "type": "string"
+        },
+        "blacklisted_pools": {
+            "type": "string"
```

Comment:
Can change type to "number" instead, so that you don't have to parseInt all the time.